metrics/ci_integrations/docs/diagrams/ci_integrations_verbose_mode_diagram.puml

```diff
@@ -0,0 +1,40 @@
+@startuml ci_integrations_verbose_mode_diagram
+
+skinparam ParticipantPadding 20
+skinparam BoxPadding 10
+
+actor Bob
+
+participant "CI Integrations"
+participant CiIntegrationsRunner
+participant SyncCommand
+participant Logger
+
+Bob -> "CI Integrations" : runs sync
+activate "CI Integrations"
+
+"CI Integrations" -> CiIntegrationsRunner: run(List<String> arguments)
+activate CiIntegrationsRunner
+
+CiIntegrationsRunner -> Logger: setup(bool verbose)
+activate Logger
+
+Logger --> CiIntegrationsRunner: void
+
+CiIntegrationsRunner -> SyncCommand: run()
+activate SyncCommand
+deactivate CiIntegrationsRunner
+
+SyncCommand -> Logger: printLog()
+
+alt runs "sync --verbose":
+    Logger --> Bob: display action logs
+end
```

Comment:
Would it be great to mention here that if the `Logger` is not initialized, the `printLog` method throws?